### DISCUSSION TITLE
How can I use  Script and CSS tabs in  Page properties window ?  Can't find any documentation for this.

### DISCUSSION THREAD

**Discussion by bhuvanesh72** [requarks/wiki#6803] [Help / Questions]
What is the use of the Script and CSS tabs in the Page properties dialog window ?
I would like to add JS code to specific pages in my wiki. Can I use these tabs for that ? If so, are there any specific codes or functions that I need to use.  I tried a few basic examples like using "alert()" and "console.log" - with no effect or result.

**Comment by NGPixel** [comment]
The scripts tab expect HTML, not direct javascript. Make sure to enclose your js in `<script type="text/javascript"></script>` tags.

**Reply by bhuvanesh72** [reply]
Thanks very much. Missed including the html type tag inside the script tags.
Will try this out and update with my experience.  regards
